pulsar-io/alluxio/pom.xml

```diff
@@ -0,0 +1,161 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>pulsar-io</artifactId>
+        <groupId>org.apache.pulsar</groupId>
+        <version>2.9.0-SNAPSHOT</version>
+    </parent>
+
+    <properties>
+        <alluxio.version>1.8.1</alluxio.version>
+        <jetty.version>9.2.16.v20160414</jetty.version>
```

Comment:
can we use the version inherited from the parent pom ?
jetty is usually reported by CVEs and we update it quite frequently

Reply:
The low vesion jetty used here is merely for test because of jar conflicts, more details can be found in https://github.com/Alluxio/alluxio/issues/8564. The latest Alluxio version may have fixed this (not sure), I can refine this in future if necessary.

Reply:
I see

Reply:
fwiw, latest alluxio 2.7.2 uses jetty 9.4.43.x (and probably can safely use 9.4.44.x, same as pulsar)

Reply:
@dlg99 Thanks for pointing out this, the new version has removed this dependency and keep the same as pulsar.